docs/sources/operations/zone-ingesters.md

```diff
@@ -0,0 +1,96 @@
+```
+ _config+:: {
+    multi_zone_ingester_enabled: true,
+    multi_zone_ingester_migration_enabled: true,
+    multi_zone_ingester_replicas: 0,
+    // These last two lines are necessary now that we enable zone aware ingester by default
+    // so that newly created cells will not be migrated later on. If you miss them you will
+    // break writes in the cell.
+    multi_zone_ingester_replication_write_path_enabled: false,
+    multi_zone_ingester_replication_read_path_enabled: false,
+  },
+```
+
+If you're not using jsonnet the new ingester statefulsets should have a label with `rollout-group: ingester`, annotation `rollout-max-unavailable: x` (put a placeholder value in, later you should set the value of this to be some portion of the statefulsets total replicas, for example in jsonnet we template this so that each statefulset runs 1/3 of the total replicas and the max unavailable is 1/3 of each statefulsets replicas), and set the update strategy to `OnDelete`.
+2.  Diff ingester and ingester-zone-a StatefulSets and make sure all config matches 
+```
+kubectl get statefulset -n loki-dev-008 ingester -o yaml > ingester.yaml
+kubectl get statefulset -n loki-dev-008 ingester-zone-a -o yaml > ingester-zone-a.yaml
+diff ingester.yaml ingester-zone-a.yaml
+```
+expected diffs are things like: creation time and revision #, the zone, fields used by rollout operator, # of replicas, anything related to kustomize/flux, and PVC for the wal since the containers don't exist yet
+3. Temporarily double max series limits for users that are using more than 50% of their current limit, the queries are as follows (add label selectors as appropriate):
+```
+sum by (tenant)(sum (loki_ingester_memory_streams) by (cluster, namespace, tenant) / on (namespace) group_left max by(namespace) (loki_distributor_replication_factor))
+>
+on (tenant) (
+max by (tenant) (label_replace(loki_overrides{limit_name="max_global_streams_per_user"} / 2.5, "tenant", "$1", "user", "(.+)"))
+)
+```
+
+```
+(sum (loki_ingester_memory_streams) by (cluster, namespace, tenant) / on (namespace) group_left max by(namespace) (loki_distributor_replication_factor)
+) / ignoring(tenant) group_left max by (cluster, namespace)(loki_overrides_defaults{limit_name="max_global_streams_per_user"}) > 0.4)
+unless on (tenant) (
+(label_replace(loki_overrides{limit_name="max_global_streams_per_user"},"tenant", "$1", "user", "(.+)")))
+```
+4. Scale up zone-aware StatefulSets until they have ⅓ of replicas each. In small cells you can do this all at once, in larger cells it might be safer to do it in chunks. The config value you need to change is `multi_zone_ingester_replicas: 6`, the value will be split across the three statefulsets. So in this case each statefulset would run 2 replicas.
+   
+   If you're not using jsonnet this is the step where you would also set the annotation `rollout-max-unavailable` to some value that is less than or equal to the # of replicas each statefulset is running.
```

Comment:
"If you're not using jsonnet" again, this is under the jsonnet heading?  Might need a separate procedure for non-jsonnet migration steps?